### PR TITLE
Duplicate verts removal

### PR DESCRIPTION
**Pull request by TfedUD**
added duplicate vert removal, passes tests and checks for previously good models and test cases: still troubleshooting revit==>doe2 issues

### CODE REVIEW

File: dragonfly_doe2/doe/utils.py [hunk 1]
```diff
@@ -26,6 +26,7 @@ def lower_left_properties(room_2d):
     """Get the vertices, boundary conditions and windows starting from lower left.
     v2 WIP
     """
+    room_2d.remove_duplicate_vertices
```
Comment (mostaphaRoudsari) [inline dragonfly_doe2/doe/utils.py:29]: Isn't this a method?
```suggestion
    room_2d.remove_duplicate_vertices()
```

**Review by mostaphaRoudsari — COMMENTED** [review]
Hi @TfedUD,

It looks like you have missed a `()` but I can be wrong. See my comment.

File: dragonfly_doe2/doe/utils.py (hunk 1 above)
Comment (TfedUD) [reply dragonfly_doe2/doe/utils.py:29]: lmao. I'm making a coffee; I should not have made that mistake
